projects/openapi-io/src/denormalizers/__tests__/__snapshots__/denormalize.test.ts.snap

```diff
@@ -0,0 +1,204 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`denormalize denormalizes shared path parameters 1`] = `
+{
+  "jsonLike": {
+    "info": {
+      "title": "some thing",
+      "version": "v0",
+    },
+    "openapi": "3.0.1",
+    "paths": {
```

Comment:
Here we've removed the shared parameter paths, and placed them onto each method. Notice how `post` does not get overriden even if we've defined something twice. 

We'll pick the _most_ specific parameter (i.e. the one on the method function)